### PR TITLE
chore(SC-940): move extensions to separate folder and cleanup

### PR DESCRIPTION
**Pull request by artall64**
- Dropped WethUnwrapper
- Dropped NonceManager and fix naming in tests
- Move ChainLink calculator to extenstions

### CODE REVIEW

File: test/SeriesNonceManager.js
Comment (ZumZoom) [inline test/SeriesNonceManager.js:1]: Rename the file as well to the `SeriesEpochManager.js`

File: contracts/helpers/SeriesNonceManager.sol
Comment (ZumZoom) [inline contracts/helpers/SeriesNonceManager.sol:1]: This file is still needed for legacy reasons. To exclude it from coverage you'll need to update the `skipFiles` sections of `.solcover.js`.

File: deploy/deploy-helpers.js
Comment (ZumZoom) [inline deploy/deploy-helpers.js:1]: This changes are irrelevant as the `SeriesEpochManager` is inherited in the `OrderMixin` so there is no need to deploy it separately. However the `SeriesNonceManager` is needed for legacy reasons, and it is deployed separately hence the need of the deploy script.

**Review by ZumZoom — CHANGES_REQUESTED** [review]

File: contracts/helpers/SeriesNonceManager.sol
Comment (artall64) [reply contracts/helpers/SeriesNonceManager.sol:1]: done

File: test/SeriesNonceManager.js
Comment (artall64) [reply test/SeriesNonceManager.js:1]: done

File: deploy/deploy-helpers.js
Comment (artall64) [reply deploy/deploy-helpers.js:1]: done

**Review by ZumZoom — APPROVED** [review]

**Pull request merged**